src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -330,6 +334,23 @@ void DlgPrefLibrary::slotRelocateDir() {
     }
 }
 
+void DlgPrefLibrary::slotSeratoMetadataExportClicked(bool checked) {
+    if (checked) {
+        if (QMessageBox::warning(this,
+                    QStringLiteral("Serato Metadata Export"),
+                    QStringLiteral(
+                            "Exporting Serato Metadata from Mixxx is highly "
+                            "experimental. There is no official documentation "
+                            "of the format. Files with Serato metadata written "
+                            "by Mixxx might crash Serato DJ, therefore caution "
```

Comment:
I do not think we should scare users by talking about Serato crashing if we have no reports of this actually happening.

Reply:
IMO it is more important to advise users to backup their music files so they can revert to a backup in case Mixxx messes up their tags.

Reply:
So yes, we should warn users, but we don't need to be dramatic about it.

Reply:
I don't think you need a backup. You can easily remove the Serato tags, e.g. `eyeD3 --remove--all-objects somefile.mp3`.

Reply:
Except if we overwrite existing Serato metadata...

Reply:
Recommending a backup is not a bad idea in any case.

Reply:
> Except if we overwrite existing Serato metadata...

On second thought that shouldn't happen because we check if parsing worked fine, and skip writing if there were any errors. But I'll add backups anyway.